compiler/luci/pass/src/CircleTypeInferencePass.cpp

```diff
@@ -41,7 +41,7 @@ bool CircleTypeInferencePass::run(loco::Graph *g)
   luci::tinf::Rule type_infer_rule;
   bool changed = false;
 
-  for (auto node : loco::postorder_traversal(loco::output_nodes(g)))
+  for (auto node : loco::all_nodes(g))
```

Comment:
From my past experience `all_nodes` may make visit to unwanted nodes.
As I see that `CircleTypeInferencePass` is not used as of now, we cannot detect any problems our test models.
We can land this change but please be cautious with future changes.

Reply:
FYI, `CircleTypeInferencePass` and [`CircleShapeInferencePass`](https://github.com/Samsung/ONE/blob/0c264a717daf9ebac489428b048c6eceead195fa/compiler/luci/pass/src/CircleShapeInferencePass.cpp#L70)will be used soon because of #5501 ..!